docs/reference/setup/jvm-options.asciidoc

```diff
@@ -0,0 +1,83 @@
+[[jvm-options]]
+=== Setting JVM options
+
+The preferred method of setting Java Virtual Machine options (including
+system properties and JVM flags) is via the `jvm.options` configuration
+file. The default location of this file is `config/jvm.options` (when
+installing from the tar or zip distributions) and
+`/etc/elasticsearch/jvm.options` (when installing from the Debian or RPM
+packages).
+
+This file contains a line-delimited list of JVM arguments following
+a special syntax:
+
+* lines consisting of whitespace only are ignored
+* lines beginning with `#` are treated as comments and are ignored
++
```

Comment:
I've added examples of each syntax line to make them more understandable

Reply:
Thanks, I think that is great.